src/data/bucket/symbol_bucket.js

```diff
@@ -344,6 +356,7 @@ class SymbolBucket implements Bucket {
         this.hasPattern = false;
         this.hasPaintOverrides = false;
         this.hasRTLText = false;
+        this.hasDebugData = false;
```

Comment:
You could make this a function that checks for the existence of the buffers instead of having a boolean that flips.